checkov/terraform/module_loading/loaders/registry_loader.py

```diff
@@ -42,7 +43,7 @@ def discover(self, module_params: ModuleParams) -> None:
 
     def _is_matching_loader(self, module_params: ModuleParams) -> bool:
         # https://developer.hashicorp.com/terraform/language/modules/sources#github
-        if module_params.module_source.startswith(("github.com", "bitbucket.org", "git::", "git@github.com")):
+        if module_params.module_source.startswith(("/", "github.com", "bitbucket.org", "git::", "git@github.com")):
```

Comment:
This change is just to prevent the below warning appearing unnecessarily for local modules when `--download-external-modules` is `true`.

```
Module {module_address} failed to load via <class 'checkov.terraform.module_loading.loaders.registry_loader.RegistryLoader'>
```